src/main/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+spring:
+  jpa:
+    database: postgresql
+    show-sql: true
+    hibernate:
+      ddl-auto: create
+  datasource:
+    url: jdbc:postgresql://localhost:8081/userapi
```

Comment:
Wise decision to not use the default ports since they can collide with a local setup